src/openrct2-ui/windows/Map.cpp

```diff
@@ -1204,7 +1204,7 @@ static CoordsXYZD place_park_entrance_get_map_position(ScreenCoordsXY screenCoor
         return parkEntranceMapPosition;
     }
 
-    parkEntranceMapPosition.z = surfaceElement->GetWaterHeight() * 8;
+    parkEntranceMapPosition.z = surfaceElement->GetWaterHeight();
```

Comment:
This deviates from the old line, but that's because the old line was incorrect (a refactor mistake in one of the Coords pull requests).